mk/config.mk

```diff
@@ -149,6 +149,12 @@ CFG_TEE_FW_MANUFACTURER ?= FW_MAN_UNDEF
 # TEE_STORAGE_PRIVATE is passed to the trusted storage API)
 CFG_REE_FS ?= y
 
+# Current REE file system Hash Tree using truncated hash size, fixing it
```

Comment:
Could you replace "Current" with "Legacy".
s/using/uses/
s/break/breaks/
s/introduced/introduce/

Maybe a bit of rephrasing:
```
CFG_REE_FS_HTREE_HASH_SIZE_COMPAT, when enabled, supports the legacy
REE FS hash tree tagging imeplementation that uses a truncated hash.
Be warned that disabling this config could break accesses to existing
REE FS content.